docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG PHP_VERSION
 
-FROM php:${PHP_VERSION}-cli
+FROM php:${PHP_VERSION}-cli-bullseye
```

Comment:
at work we had a lot of subtle problems coming from the new `bookworm` release of debian

foundry's CI was failing as well for unknown reasons, so I preferred to stick to bullseye for now